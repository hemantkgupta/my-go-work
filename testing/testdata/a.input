"package p\n\nconst _ = 0000000123i\n"