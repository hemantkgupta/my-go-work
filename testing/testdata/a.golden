"package p\n\nconst _ = 123i\n"